build.sbt

```diff
@@ -32,28 +34,66 @@ lazy val allSettings = Seq(
         <name>Ólafur Páll Geirsson</name>
         <url>https://geirsson.com</url>
       </developer>
+      <developer>
+        <id>jonas</id>
+        <name>Jonas Fonseca</name>
+        <url>https://github.com/jonas</url>
+      </developer>
     </developers>
 )
 
 lazy val example = project
 
-lazy val metadoc = crossProject
-  .in(file("metadoc"))
+lazy val protobufSettings = Seq(
+  PB.targets.in(Compile) := Seq(
+    scalapb.gen(
+      flatPackage = true // Don't append filename to package
+    ) -> sourceManaged.in(Compile).value./("protobuf")
```

Comment:
Great, I tried `PB.deleteTargetDirectory in Compile := false` but that didn't work either:
```
[info] Compiling 7 Scala sources to /opt/metadoc/metadoc-cli/target/scala-2.12/classes...
[error] /opt/metadoc/metadoc-cli/target/scala-2.12/src_managed/main/sbt-buildinfo/BuildInfo.scala:6: BuildInfo is already defined as case class BuildInfo
[error] case object BuildInfo {
[error]             ^
[error] one error found
[error] (cli/compile:compileIncremental) Compilation failed
```

Eventually, we need to move the protobuf code generation to a shared project so we can decode the protobuf files in the `js` project.